### ISSUE TITLE
Different requests for files

### ISSUE THREAD

**Issue by kovalevskyartur** [Wildhoney/ngDroplet#17]
Is it possible to make request for each file? If i set useArray to false then Droplet make one request only for one of the files (not support multiple files) and all.

I think this feature very important and useful

**Comment by Wildhoney** [comment]
`useArray` will still send only one request with all of the files, but [it's the name it gives to the file property](https://github.com/Wildhoney/ngDroplet/blob/master/components/ngDroplet.js#L553) as I noticed a discrepency with the way different frameworks handle multiple files. Some frameworks require the `[]` notation for the `files` property, whereas other frameworks/libraries don't.

Do you want to make individual requests per each file? I can look into implementing that, as currently it's not supported.

**Comment by kovalevskyartur** [comment]
Yes, I'm looking for individual requests per each file. It will be great if you will make this feature.

**Comment by rodriguezmanu** [comment]
You can do something like that:

```
scope.uploadFiles = function uploadFiles() {
                    // Reset...
                    $scope.isError = false;

                    var httpRequest   = new $window.XMLHttpRequest(),
                        formData      = new $window.FormData(),
                        queuedFiles   = $scope.filterFiles($scope.FILE_TYPES.VALID),
                        fileProperty  = $scope.options.useArray ? 'file[]' : 'file',
                        requestLength = $scope.getRequestLength(queuedFiles),
                        deferred      = $q.defer();

$angular.forEach(queuedFiles, function forEach(model) {
.....
)};
```

**Comment by rodriguezmanu** [comment]
@kovalevskyartur I've created a pull request to do that, I hope help you. Here https://github.com/Wildhoney/ngDroplet/pull/53/
